gulpfile.js

```diff
@@ -13,7 +12,7 @@ const gulpif = require('gulp-if');
 const gutil = require('gulp-util');
 const imagemin = require('gulp-imagemin');
 const minifycss = require('gulp-minify-css');
-const neat = require('node-neat');
+const normalize = require('node-normalize-scss');
```

Comment:
I'm not sure either, but I really don't want to dig around in the gulpfile. PRs welcome?